content/en/docs/tasks/manage-kubernetes-objects/kustomization.md

```diff
@@ -749,7 +749,7 @@ not recommended to hard code the Service name in the command argument. For this
 
 ```shell
 # Create a deployment.yaml file
-cat <<EOF > deployment.yaml
+cat <<\EOF > deployment.yaml
```

Comment:
Ah.
Try this, it's more conventional, and the comment should explain why the quoting is there.
```suggestion
# Create a deployment.yaml file (quoting the here doc delimiter)
cat <<'EOF' > deployment.yaml
```

Reply:
Thank you. This fix is ​​fine.